src/index.js

```diff
@@ -5,6 +5,8 @@ import React, { useState } from "react";
 import { createRoot } from "react-dom/client";
 import reportWebVitals from "reportWebVitals";
 
+require('dotenv').config()
```

Comment:
You don't need this. It will fail the build.

Reply:
Done.